log10/load.py

```diff
@@ -310,8 +342,8 @@ def wrapper(*args, **kwargs):
                 "session_id": sessionID,
                 "tags": global_tags,
             }
-            res = post_request(completion_url + "/" + completionID, log_row)
-            raise e
+            if not got_log10_exception:
+                res = post_request(completion_url + "/" + completionID, log_row)
```

Comment:
```suggestion
                res = post_request(completion_url + "/" + completionID, log_row)
                if res is None or res.status_code != 200:
                    logger.error(f"LOG10: failed to insert in log10: {log_row}.")
                    got_log10_exception = True
```
Here may fail and we should log the error.